### PR TITLE
what's next for network plugin

### PR DESCRIPTION
**Pull request by kundan2707**
Updated what's next section in network plugin page

### CODE REVIEW

**Comment by kundan2707** [comment]
fixes: #42103

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -172,3 +172,9 @@ metadata:
 
 ## {{% heading "whatsnext" %}}
 
+* Learn about [Network Policies](/docs/concepts/services-networking/network-policies/) using network
+  plugins
+* Learn about [Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking/)
+  with network plugins
+* Learn about the [Troubleshooting CNI plugin-related errors](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors/)
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:179]: We use site-relative links where feasible.

```suggestion
* Learn about [Cluster Networking](/docs/concepts/cluster-administration/networking/)
  with network plugins
* Learn about the [Troubleshooting CNI plugin-related errors](/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors/)
```

**Comment by kundan2707** [comment]
@dipesh-rawat  resolved as per your suggestion.

**Comment by kundan2707** [comment]
@dipesh-rawat  incorporated review comments

**Comment by kundan2707** [comment]
@thockin @dcbw can you please have a look?

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 2]
```diff
@@ -172,3 +172,9 @@ metadata:
 
 ## {{% heading "whatsnext" %}}
 
+* Learn about [Network Policies](/docs/concepts/services-networking/network-policies/) using network
+  plugins
+* Learn about [Cluster Networking](/docs/concepts/cluster-administration/networking/)
+  with network plugins
```
Comment (tengqm) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:178]:
```suggestion
* Learn more about [Cluster Networking](/docs/concepts/cluster-administration/networking/)
* Learn more about [Network Policies](/docs/concepts/services-networking/network-policies/)
```

**Pull request merged**